src/main/java/org/broadinstitute/hellbender/tools/exome/AllelicCNV.java

```diff
@@ -116,7 +116,7 @@
             shortName = ExomeStandardArgumentDefinitions.ALLELIC_PON_FILE_SHORT_NAME,
             optional = true
     )
-    protected File allelicPONFile;
+    protected File allelicPoNFile;
```

Comment:
Should this be `allelicPonFile`? I don't know how much we follow the Google conventions...